test/support/coverage.rb

```diff
@@ -0,0 +1,43 @@
+module Coverage
```

Comment:
I quite like the `module` - `module_function` approach - I can change it to a class with class functions if that suits the style of the repo better, do you want me to?